services/azure-devops/azure-devops-release.tester.js

```diff
@@ -13,6 +13,13 @@ t.create('release status is succeeded')
     message: isBuildStatus,
   })
 
+t.create('never deployed')
+  .get('/totodem/8cf3ec0e-d0c2-4fcd-8206-ad204f254a96/1/4.json')
+  .expectBadge({
+    label: 'deployment',
+    message: 'never deployed',
```

Comment:
Are we confident that this will always be in a "never deployed" state?

Reply:
We are as I setup this stage for that matter: https://dev.azure.com/totodem/Shields.io.

Reply:
Awesome!